.drone.yml

```diff
@@ -0,0 +1,51 @@
+kind: pipeline
+type: kubernetes
+name: Deployment Stable
+
+steps:
+- name: docker-build
+  image: docker
+  pull: if-not-exists
+  environment:
+    DOCKER_REGISTRY: 558830342743.dkr.ecr.us-east-1.amazonaws.com
+  commands:
+    - docker build --network=host -t $$DOCKER_REGISTRY/healthcheck/webtests/checkout:latest -f ./dockerfiles/stable/Dockerfile .
+  volumes:
+    - name: dockersock
+      path: /var/run
+- name: publish
+  image: 558830342743.dkr.ecr.us-east-1.amazonaws.com/reliability/jenkins/docker:latest
```

Comment:
hmm how does this works? why do we need jenkins?

Reply:
it doesn't include jenkins, it is the image the SRE team has created that includes docker and a script to authenticate to AWS (you can see it in use in the `commands` section)

Reply:
ah I see, so this image is just placed weirdly at the `jenkins` directory, right?

Reply:
yep